release/models/system/openconfig-system-logging.yang

```diff
@@ -473,6 +479,142 @@ module openconfig-system-logging {
     }
   }
 
+  grouping logging-file-config {
+    description
+      "Configuration data for logfile";
+
+    leaf filename-prefix {
+      type string {
+        length 0..255;
+      }
+      description
+        "filename prefix. It is expected that implementation may append to it timestamp, serial-number or other type of identifier";
+    }
+
+    leaf path {
+      type string {
+        length 0..255;
+      }
+      description
+        "path to the folder where logfile is stored. May include drive identifier is implementation requires";
+    }
+
+    leaf count {
+      type uint32;
+      default 1;
+      description
+        "Used for logfile rotation. maximum number of files of given filename-prefix stored. Default value: 1. ";
+    }
+
+    leaf max-size {
+      type uint32;
+      default 1000;
+      description
+        "Maximum size logfile may grow to. When logfile reach this size it need to be closed, save, and new file open or future log storage. If needed oldest logfile of same prefix shall be deleted to";
```

Comment:
What units should be used?  Probably bytes?  

Also, per https://www.openconfig.net/docs/guides/style_guide/, line length should be no more than 70 characters long.